src/main/java/stream/flarebot/flarebot/util/MessageUtils.java

```diff
@@ -153,6 +152,16 @@ public static Message sendWarningMessage(String message, MessageChannel channel)
                 .complete();
     }
 
+    public static Message sendSuccessMessage(String message, TextChannel channel) {
```

Comment:
Why have these? I see no point.

Reply:
We send a lot of success and info messages and we always write the same code, I plan to move over to helper messages a lot more